woocommerce/woocommerce
# Add 47 Kenya counties

### All Submissions:






### Changes proposed in this Pull Request:



Closes https://github.com/woocommerce/woocommerce/issues/26722. 

Currently, we don't have any regions added for Kenya in WooCommerce which makes it not a good experience for merchant when they are looking to set up shipping zones for Kenya. Setting up shipping zones based on zip codes is not sufficient enough. 

This PR adds 47 Kenya counties as per the latest release version 37 of UNICODE CLDR that is being adopted in WooComerce: 

https://github.com/unicode-org/cldr/blob/release-37/common/subdivisions/en.xml#L2549-L2596

### How to test the changes in this Pull Request:

1. Before testing this PR, note in `WooCommerce / Settings / Shipping / Shipping zones` if you search for Kenya, no states/regions will come up:

![kenya1](https://user-images.githubusercontent.com/19143190/84083043-e1727600-a9ae-11ea-93f3-7d76b650d9e3.jpg)

As per @claudiosanches's recommendations, I did the following:

- the latest release number 37 of UNICODE CLDR was used;
- county codes such as `ke01` were converted to `KE01`;
- `KE01` was used instead of `KE-01` since we no longer use ISO. 

2. After checking out the branch of this PR, 47 Kenya counties should come up:

![kenya2](https://user-images.githubusercontent.com/19143190/84083133-0d8df700-a9af-11ea-93b3-1c848676e611.jpg)

3. Also, it will be good to test that counties come up on the Checkout page once Kenya is selected as a Billing/Shipping country:

![kenya3](https://user-images.githubusercontent.com/19143190/84083315-62317200-a9af-11ea-9217-d733aa546600.jpg)

### Other information:




### Changelog entry

> Enhancement - Add 47 Kenya counties.
